src/python/grpcio/grpc/_channel.py

```diff
@@ -374,13 +376,20 @@ def details(self):
                 self._state.condition.wait()
             return _common.decode(self._state.details)
 
+    def debug_error_string(self):
+        with self._state.condition:
+            while self._state.debug_error_string is None:
```

Comment:
Is `self._state.debug_error_string` ever `None`?

Reply:
Now it can be :)